gui/src/app/components/status/status.js

```diff
@@ -43,6 +51,8 @@ class Status extends React.Component {
     // Navigate to the app in the inspector, clearing the state so that the
     // app overview is shown.
     this.props.changeState(this._generateApplicationClickState(appId));
+    this.analytics.addCategory('Application').sendEvent(
```

Comment:
It will be difficult to know "Application" is part of the Status component. Could this be called "Application status" or something?

Reply:
This will be have "Status" add from the call above (`this.analytics = this.props.analytics.addCategory('Status');`) so the final path would be "Status > Application". Same for the events below.